website/blog/2020-02-31-version-60.md

```diff
@@ -0,0 +1,61 @@
+
+## Lean Core Removals
+
+**WebView** and **NetInfo** were previously extracted into separate repositories, and in 0.60 we’ve finished migrating them out of the React Native repository. Additionally, in response to community feedback about new App Store policy, **Geolocation** has been extracted. If you haven’t already, complete your migration by depending on [react-native-webview](https://github.com/react-native-community/react-native-webview), [@react-native-community/netinfo](https://github.com/react-native-community/react-native-netinfo), and [@react-native-community/geolocation](https://github.com/react-native-community/react-native-geolocation). Maintainers have made more than 100 commits to these repositories since extraction; we’re excited to see the community’s support!
+
+## Native Modules are now Autolinked
+
+The team working on [the cli](https://github.com/react-native-community/cli) has introduced major improvements to native module linking called [autolinking](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md)! With it, the `react-native link` command is replaced by automatic native platform package linking, with no additional steps required. At the same time, the team overhauled the linking process in general. Let us know what you think using the hashtag [#autolinkingInRN](https://twitter.com/intent/tweet?hashtags=autolinkingInRN).
+
+## Upgrade Helper
+
+[@lucasbento](https://github.com/lucasbento), [@pvinis](https://github.com/pvinis), [@kelset](https://github.com/kelset), and [@watadarkstar](https://github.com/watadarkstar) have built a great tool called [Upgrade Helper](https://react-native-community.github.io/upgrade-helper/) to make the upgrade process simpler. This helps React Native users with brownfield apps or complex customizations more quickly see what's changed between versions. Take a look at the [upgrade from 0.59.9 to 0.60.0](https://react-native-community.github.io/upgrade-helper/?from=0.59.9&to=0.60.0) and try it out today for your upgrade path!
+
+![Upgrade Helper cleanly and easily shows the changes to migrate to a different version of React Native](/react-native/blog/assets/0.60-upgrade-helper.png)
+
+## A Note to Library Maintainers
+
+Changes for AndroidX will almost certainly require updates to your library, so be sure to bring support soon. If you're not able to upgrade yet, consider checking your library against the jetifier to confirm users are able to patch your library at build time.
+
+Depending on how your library was previously integrated, you may also need to make some changes. Check the [dependencies](https://github.com/react-native-community/cli/blob/master/docs/dependencies.md) guide from the CLI for information on how to define your dependency interface.
```

Comment:
We'll need to create a single migration guide and link it here. Cc @grabbou

Reply:
@thymikee maybe I can help with that; let me know what you have in mind, and I'll try to help Sunday.

Reply:
@turnrye 	https://github.com/react-native-community/cli/pull/450

Reply:
Oh, we also need to ask them to adopt their configs and readmes to utilize autolinking, docs: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md